frontend/src/app/patients/Components/PersonForm.tsx

```diff
@@ -552,6 +552,7 @@ const PersonForm = (props: Props) => {
           legend={t("patient.form.demographics.gender")}
           hintText={t("patient.form.demographics.genderHelpText")}
           name="gender"
+          required={view !== PersonFormView.PXP}
```

Comment:
Should this field be required during self-registration as well?

Reply:
I believe it currently is required for self-registration as it is written, as this will evaluate to true for `personSchema` and `selfRegistrationSchema`.

I also had to update the selfRegistration test in order to get them to pass as this is currently written.

```
[PersonFormView.APP]: personSchema,
    [PersonFormView.PXP]: personUpdateSchema,
    [PersonFormView.SELF_REGISTRATION]: selfRegistrationSchema,
```

Reply:
Oh! I thought self-registration _was_ the PXP view. In that case I have no clue what it actually is 🙈